packages/components/src/Popover/Popover.test.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { useRef } from "react";
+import {
+  RenderResult,
+  act,
+  cleanup,
+  fireEvent,
+  render,
+  waitFor,
+} from "@testing-library/react";
+import { Popover } from ".";
+import { PopoverProps } from "./Popover";
+
+afterEach(cleanup);
+
+let rendered: RenderResult;
+const title = "Test Title";
+const content = "Test Content";
+
+const PopoverTestComponent = (props: PopoverProps) => {
+  const checkboxRef = useRef();
+  return (
+    <>
+      <div ref={checkboxRef}></div>
+      <Popover {...props} attachTo={checkboxRef}>
+        {props.children}
+      </Popover>
+    </>
+  );
+};
+
+it("should render a Popover with a title, content, and dismiss button", async () => {
+  const handleClose = jest.fn();
+
+  await act(async () => {
+    rendered = render(
+      <PopoverTestComponent
+        attachTo={undefined}
+        title={title}
+        open={true}
+        onRequestClose={handleClose}
+      >
+        {content}
+      </PopoverTestComponent>,
+    );
+  });
+
+  const { getByText, queryByTestId, container } = rendered;
+
+  expect(getByText(title)).toBeTruthy();
```

Comment:
I think, rather than using `getBy..` we can use `queryBy..` since that will return `null` if not present and subsequently not be "truthy". (Rather than `getBy..` which will throw an error)